lib/y2_logs_helper.pm

```diff
@@ -101,12 +101,19 @@ sub accept_license {
 }
 
 sub verify_license_translations {
-    return if (is_sle && get_var("BETA") || check_var('VIDEOMODE', 'text'));
+    return if (is_sle && get_var("BETA"));
     my $current_lang = 'english-us';
     for my $lang (split(/,/, get_var('EULA_LANGUAGES')), 'english-us') {
         wait_screen_change { send_key 'alt-l' };
-        assert_and_click "license-language-selected-$current_lang";
-        wait_screen_change { type_string(substr($lang, 0, 1)) };
+        # in textmode only arrow navigation is possible
+        if (get_var('VIDEOMODE') =~ 'text') {
+            send_key_until_needlematch("license-language-selected-english-us", 'up', 60);
```

Comment:
This looks wrong to me. `english-us` is default language we check, should we use same needle as for qt, namely: `license-language-selected-$current_lang`?

Reply:
In ncurse mode, i have to navigate back in the top of the list and start checking for the $lang. the first entry is english-us. The current_lang in the second iteration is korean in this case. So fails. i tried to avoid this but it ends with more code and make it complicated with no reason in my point of view.

Reply:
Ah, I see. Thanks for the clarification.